source-map.d.ts

```diff
@@ -1,98 +1,115 @@
-export interface StartOfSourceMap {
-    file?: string;
-    sourceRoot?: string;
-}
+declare module 'source-map-js' { 
```

Comment:
The `declare module` syntax is only for stubbing third-party modules. To declare modules in the current package, you should just write a separate `.d.ts` file for each `.js` file. This fixes TS2306 “File `…/source-map.d.ts` is not a module” errors.

- #16